### PR TITLE
Fixes Bowling Bash behavior

### PR DESCRIPTION
**Pull request by aleos89**
* **Addressed Issue(s)**: #5489

* **Server Mode**: Renewal

* **Description of Pull Request**: 
  * Removes the gutterline feature from Bowling Bash in renewal mode.
  * Skill will now behave as a normal 5x5 AoE.
  * Fixes knockback getting removed during battle calculation.
Thanks to @humanwizzard!

### CODE REVIEW

**Comment by humanwizzard** [comment]
hit count increase depending on the number of targets with 2h sword: not working = always 2 hits 

knockback should be:
lv 1 ~ 2 = 1
lv 3 ~ 4 = 2
lv 5 ~ 6 = 3
lv 7 ~ 8 = 4
lv 9 ~ 10 = 5

**Comment by humanwizzard** [comment]
knockback works strange now:
1 ~ 3 targets = knockback works
4 ~ 7 targets = no knockback
8 ~ 11 targets = knockback works
12 ~ 15 targets = no knockback
and so on

**Comment by humanwizzard** [comment]
knockback works fine now but hit count got broken:
4 ~ 5 targets = 2 hit
6 ~ 7 targets = 3 hit

**Comment by humanwizzard** [comment]
knockback works 2 times (2 ~ 10 cells) on all except one target and hit count works only on that target

File: src/map/skill.cpp [hunk 1]
```diff
@@ -5272,7 +5272,20 @@ int skill_castend_damage_id (struct block_list* src, struct block_list *bl, uint
 			skill_attack(skill_get_type(skill_id), src, src, bl, skill_id, skill_lv, tick, flag);
 		break;
 
+#ifdef RENEWAL
 	case KN_BOWLINGBASH:
+		if (flag & 1) {
+			skill_attack(skill_get_type(skill_id), src, src, bl, skill_id, skill_lv, tick, (skill_area_temp[0]) > 0 ? SD_ANIMATION | skill_area_temp[0] : skill_area_temp[0]);
+			skill_blown(src, bl, skill_get_blewcount(skill_id, skill_lv), -1, BLOWN_NONE);
+			skill_area_temp[0] = 0;
```
Comment (Daegaladh) [inline src/map/skill.cpp:5280]: This line makes the recursive invocation not working properly

**Review by Daegaladh — CHANGES_REQUESTED** [review]

File: src/map/skill.cpp (hunk 1 above)
Comment (Daegaladh) [inline src/map/skill.cpp:5280]: This line makes recursive invocation not work properly

**Review by Daegaladh — CHANGES_REQUESTED** [review]

**Review by Daegaladh — APPROVED** [review]

**Pull request merged**